packages/key-management/src/types.ts

```diff
@@ -38,10 +38,26 @@ export enum KeyRole {
   DRep = 3
 }
 
+export enum MultiSigKeyRole {
+  Payment = 0,
+  Stake = 2
+}
+
+export enum KeyPurpose {
+  INDIVIDUAL = 1852,
+  MULTI_SIG = 1854
+}
+
 export interface AccountKeyDerivationPath {
   role: KeyRole;
   index: number;
 }
+
+export interface MultiSigAccountKeyDerivationPath {
```

Comment:
same here, we dont need a special case for this, you can use `AccountKeyDerivationPath`

Reply:
Removed on fac71bd